app/policies/admin/site_notice_policy.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::SiteNoticePolicy < ApplicationPolicy
+
+  class Scope < Scope
+    def resolve
+      all.order('updated_at desc')
```

Comment:
Done.  I removed the scope (since it was just returning all which is the default) and updated the commented out scope in the controller:

    #@all_notices = policy_scope(Admin::SiteNotice).order('updated_at desc')